docs/developer-guide/testing.md

```diff
@@ -79,6 +79,6 @@ However, as a word of caution, be aware that existing integration tests will
 start using the new programs immediately, so this should only be done if the
 modified program is backward compatible.
 
-## Kubernetes Integration Tests
+## Kubernetes Operator Unit and Integration Tests
```

Comment:
If you change the heading, need to update the link in https://github.com/bpfman/bpfman/blob/main/CONTRIBUTING.md line 219